package.json

```diff
@@ -97,6 +97,7 @@
     "event-target-shim": "^5.0.1",
     "fbjs-scripts": "^1.1.0",
     "hermes-engine": "~0.6.0",
+    "hermes-engine-darwin": "0.6.0",
```

Comment:
Not sure what's the latest versioning pattern, but I followed what we do for `hermes-engine` already. Note that `hermes-engine` could potentially be renamed to `hermes-engine-android`.